db/fixtures/ae_datastore/ManageIQ/Service/Provisioning/StateMachines/ServiceProvision_Template.class/__class__.yaml

```diff
@@ -13,7 +13,7 @@ object:
   schema:
   - field:
       aetype: state
-      name: pre1
+      name: pre0
```

Comment:
@billfitzgerald0120 @tinaafitz 
Should we be calling this state sequencer instead of pre0